src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.js

```diff
@@ -2,31 +2,52 @@ import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import { Link } from 'react-router-dom';
 
-export const BannerUpgrade = ({ currentPlan, currentPlanList }) => {
-  const getBannerInfo = (type) => {
+export const BannerUpgrade = ({ sessionPlan, currentPlan, currentPlanList }) => {
+  const getUpgradeInfo = (type) => {
     const bannerInfo = { messageId: `plan_calculator.banner_for_${type.replace('-', '_')}` };
+    const suggestionInfo = {
+      messageId: `plan_calculator.suggestion_for_${type.replace('-', '_')}`,
+    };
+
     switch (type) {
       case 'prepaid':
       case 'subscribers':
         return {
-          ...bannerInfo,
-          link: `/plan-selection/${currentPlan.featureSet}/monthly-deliveries`,
+          banner: {
+            ...bannerInfo,
+            link: `/plan-selection/${currentPlan.featureSet}/monthly-deliveries`,
+          },
+          suggestion: {
+            ...suggestionInfo,
+            link: '#',
```

Comment:
Here we will have the form for upgrade right?

Reply:
Yes, that's right 😉